general/scans/motion.py

```diff
@@ -219,31 +219,8 @@ def get_motion(motion_or_block_name):
     if isinstance(motion_or_block_name, Motion):
         motion = motion_or_block_name
     elif isinstance(motion_or_block_name, (str, text_type)):
-        motion = BlockMotion(motion_or_block_name, get_units(motion_or_block_name))
+        motion = BlockMotion(motion_or_block_name, g.cget(motion_or_block_name)["units"])
```

Comment:
Can this not use your new method in genie python? I would expect this call to look something like g.get_block_units(block)